pkg/compute/bidder.go

```diff
@@ -36,7 +34,6 @@ type Bidder struct {
 	usageCalculator capacity.UsageCalculator
 	executor        Executor
 	callback        Callback
-	getApproveURL   func() *url.URL
```

Comment:
Is this feature being removed, or just moved somewhere else?

Reply:
feature removed ages ago